src/components/App/components/UserInput/UserInput.js

```diff
@@ -68,15 +70,22 @@ function UserInput(props) {
                 inputProps={{
                   id: 'font-size',
                   type: 'number',
-                  min: MIN_FONT_SIZE,
-                  max: MAX_FONT_SIZE,
+                  min: MIN_FONT_SIZE_IN_PX,
+                  max: MAX_FONT_SIZE_IN_PX,
```

Comment:
Now, that we have `IN_PX`, this doesn't look right.
We made the variable name more clear, and it highlighted a logical issue.
The question I ask myself here is, why the `min` and `max` are `IN_PX`. Why not `IN_PT`?
Indeed, if you change the unit to `pt`, you'll be able to go up to 36pt, which, again, reaches the area where changing the font size is not reflected in the UI (27-36pt looks the same because they all above our max which is 36px).
Now we have an interaction issue :)
What do you think should happen when you set `36px`, and then toggle to `pt`?
Should it jump to `24pt` because it's our maximum? (equivalent to 36px)
This jump from 36 to 24 can feel weird, but at least we don't allow users to get to a bad state (>24pt).
What do you think?